src/tests/ftest/cart/tests_common.h

```diff
@@ -51,11 +51,12 @@ struct test_options {
 	int		num_attach_retries;
 	bool		is_server;
 	bool		assert_on_error;
+	volatile int	shutdown;
```

Comment:
(style)  Use of volatile is usually wrong: see Documentation/volatile-considered-harmful.txt

Reply:
And not using volatile when it should be used is also wrong.  This decision is yours, I would use it.